layouts/article.html

```diff
@@ -15,6 +15,11 @@
         <div class="markdown-body document-content">
           <% section = lookup_section('toc', @item[:title]) %>
           <h1><%= section << ' ' << @item[:title] if section %></h1>
+          <h1 class="improve">
+            <a href="<%= "https://github.com/atom/flight-manual.atom.io/edit/master/content" + @item.identifier %>">
```

Comment:
Or maybe instead of the `|` divider, we could add bit more space in between?

![screen shot 2016-03-25 at 12 55 43 pm](https://cloud.githubusercontent.com/assets/378023/14037566/eaab95da-f288-11e5-8526-d8f817f3609c.png)

```html
<h1><%= section << ' ' << @item[:title] if section %>
  <a class="improve-link" href="<%= "https://github.com/atom/flight-manual.atom.io/edit/master/content" + @item.identifier %>"><span class="octicon octicon-pencil"></span>  Improve this page</a>
</h1>
```

```scss
// in /assets/stylesheets/app/_flight_manual.scss
.improve-link {
  margin-left: 1em;
}
```